python/python/ert_gui/simulation/models/base_run_model.py

```diff
@@ -46,11 +46,11 @@ def __init__(self, queue_config, phase_count=1):
         self._queue_config = queue_config
         self._job_queue = None
         self.realization_progress = {}
-        self.initial_realizations_mask = None
-        self.completed_realizations_mask = None
+        self.initial_realizations_mask = []
+        self.completed_realizations_mask = []
```

Comment:
completed_realization mask is contained in run_context and is not strictly needed anymore. Should be revisited when a more comprehensive debugging tool is created.